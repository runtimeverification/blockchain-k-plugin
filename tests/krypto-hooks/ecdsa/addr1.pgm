#addrFromPrivateKey("0000000000000000000000000000000000000000000000000000000000000001")
