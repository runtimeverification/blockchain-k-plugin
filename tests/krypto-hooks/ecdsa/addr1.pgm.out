<k>
  721457446580647751014191829380889690493307935711 ~> .
</k>
